.travis.yml

```diff
@@ -2,8 +2,6 @@ language: node_js
 node_js:
   - node
 
-sudo: required
```

Comment:
In most cases we're not limited by the memory of the container, 4GB will be enough. But the boot time is apparently significantly quicker if it's disabled.